service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/aws/sageMakerNotebook/AwsSageMakerNotebookFlightMiscTest.java

```diff
@@ -0,0 +1,122 @@
+package bio.terra.workspace.service.resource.controlled.cloud.aws.sageMakerNotebook;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.common.exception.NotFoundException;
+import bio.terra.stairway.FlightDebugInfo;
+import bio.terra.stairway.FlightStatus;
+import bio.terra.stairway.StepStatus;
+import bio.terra.workspace.common.fixtures.ControlledAwsResourceFixtures;
+import bio.terra.workspace.common.utils.AwsUtils;
+import bio.terra.workspace.generated.model.ApiAwsSageMakerNotebookCreationParameters;
+import bio.terra.workspace.generated.model.ApiJobControl;
+import bio.terra.workspace.service.iam.model.ControlledResourceIamRole;
+import bio.terra.workspace.service.resource.exception.ResourceNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.services.sagemaker.model.NotebookInstanceStatus;
+
+// Misc. connectedPlus tests
+@Tag("aws-connected-plus")
+public class AwsSageMakerNotebookFlightMiscTest extends BaseAwsSageMakerNotebookFlightTest {
+
+  @Test
```

Comment:
Yes, it is because of the runtime.
Since 'do' is more critical and is part of regular success flow, it would be run as part of the connectedTest (PR checks, etc). 

Undo flow is relatively rare assuming creation via do goes well (critical nevertheless). Hence made it part of the connectedPlus test which is part of Nightly run. Adding this to the the connected Test will make PR checks runs over an hr easily.